misc-tools/import-contest

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+'''
+import-contest -- Convenience script to import a contest (including metadata, teams and
+problems) via the command line. Supports reading credentials from ~/.netrc. See also
+https://www.domjudge.org/docs/manual/main/import.html
+
+Part of the DOMjudge Programming Contest Jury System and licensed
+under the GNU GPL. See README and COPYING for details.
+'''
+
+import fileinput
+import json
+import os.path
+import requests
+import requests.utils
+import sys
+from typing import List
+import yaml
+
+headers = {
+    "user-agent": f"domjudge-import-contest ({requests.utils.default_user_agent()})"}
```

Comment:
nit: be consistent about using `'` or `"` for all strings (I would use the former)

Reply:
I thought I used `"` everywhere but apparently not. If you prefer `'` I will change them.